packages/libs/src/sdk/api/generated/full/models/CollectionPurchasableContentFull.ts

```diff
@@ -0,0 +1,99 @@
+/* tslint:disable */
+/* eslint-disable */
+// @ts-nocheck
+/**
+ * API
+ * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
+ *
+ * The version of the OpenAPI document: 1.0
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+import { exists, mapValues } from '../runtime';
+import type { PlaylistFull } from './PlaylistFull';
+import {
+    PlaylistFullFromJSON,
+    PlaylistFullFromJSONTyped,
+    PlaylistFullToJSON,
+} from './PlaylistFull';
+import type { PurchasableContentFull } from './PurchasableContentFull';
+import {
+    PurchasableContentFullFromJSON,
+    PurchasableContentFullFromJSONTyped,
+    PurchasableContentFullToJSON,
+} from './PurchasableContentFull';
+
+/**
+ * 
+ * @export
+ * @interface CollectionPurchasableContentFull
+ */
+export interface CollectionPurchasableContentFull extends PurchasableContentFull {
+    /**
+     * 
+     * @type {string}
+     * @memberof CollectionPurchasableContentFull
+     */
+    contentType: CollectionPurchasableContentFullContentTypeEnum;
+    /**
+     * 
+     * @type {PlaylistFull}
+     * @memberof CollectionPurchasableContentFull
+     */
+    item: PlaylistFull;
```

Comment:
Just tying this to the comment above about playlist models. The generated code for this type requires a nested `tracks` array. So I would verify that's actually returned.